explore-analyze/query-filter/languages/esql-cross-clusters.md

```diff
@@ -470,10 +470,27 @@ FROM my-index-000001,cluster*:my-index-*,cluster_three:-my-index-000001
 
 
 ## Optional remote clusters [ccq-skip-unavailable-clusters]
+```{applies_to}
+stack: ga 9.1
```

Comment:
```suggestion
stack: preview 9.0, ga 9.1
```
this isn't brand new and the docs now cover all 9.x versions, so the reader might be on 9.0 or 9.1 so we need to maintain lifecycle information

Reply:
but if the entire page is marked `preview 9.0, GA 9.1`, you might not need this section-level tag at all?

Reply:
@smalyshev in this case you should use [tabs](https://elastic.github.io/docs-builder/syntax/tabs/) to house the 9.0 and 9.1 differences:


``````markdown

::::{tab-set}

:::{tab-item} 9.0
This is where the content for tab #1 goes.
:::

:::{tab-item} 9.1
This is where the content for tab #2 goes.
:::

::::

Reply:
Oh interesting. I'll try that.